src/services/pool/decorators/pool.multicaller.ts

```diff
@@ -51,8 +50,6 @@ export class PoolMulticaller {
     const multicaller = new this.MulticallerClass();
 
     this.pools.forEach(pool => {
-      pool = removeBptFrom(pool);
```

Comment:
This may be a case where we need to actually remove the address because it effects the onchain calls below?

Reply:
yes, in `if (isDeep(pool)) {` is used tokenList. I removed pre-minted bpt there

Reply:
But there are no more changes in this file? And look further down here: https://github.com/balancer-labs/frontend-v2/blob/f50f16d27cead5f83ef049e73ec2c5d036195319/src/services/pool/decorators/pool.multicaller.ts#L116 This is expecting to not have the bpt token in the list.

Reply:
yeah, I mean I've done it locally, sorry, just not pushed it yet

Reply:
done

Reply:
I still don't see other changes in this file? The line I referenced above will cause issues if BPT isn't removed here.

Reply:
it seems that this line you mentioned above doesn't exist anymore 🤔

Reply:
This is what the pool.multicaller.ts looks like on `develop` now: https://github.com/balancer-labs/frontend-v2/blob/develop/src/services/pool/decorators/pool.multicaller.ts

This file in this PR looks very different and still contains all the linear pool calls. You will need to sync with develop first.

Reply:
I created new pr: #2639